resources/var/tmp/resources/init.groovy.d/scripts/init030installplugins.groovy

```diff
@@ -55,6 +55,21 @@ for (def shortName : plugins){
   }
 }
 
+// Make sure CAS-Plugin version is at least 1.4.3 to work with Jenkins 2.150.2 and following
+def minimalCasPluginVersion = "1.4.3"
+def currentCasPlugin = jenkins.getPluginManager().getPlugin('cas-plugin');
+if (currentCasPlugin == null) {
```

Comment:
Can this happen? We just installed the plugins.
Maybe we could remove the cas-plugin(before installing all) if it has a to old version. This would deduplicate code.